codegen/src/main/java/software/amazon/awssdk/codegen/poet/waiters/BaseWaiterClassSpec.java

```diff
@@ -69,6 +70,11 @@
  */
 public abstract class BaseWaiterClassSpec implements ClassSpec {
 
+    public static final String FAILURE_MESSAGE_FORMAT_FOR_PATH_MATCHER = "A waiter acceptor with the matcher %s matched the "
```

Comment:
Suggesting the following:
```
An error waiter acceptor with the matcher (%s) was matched on parameter (%s=%s) and 
transitioned the waiter to failure state.
```

Reply:
Was wondering why `error waiter acceptor` and not just `waiter acceptor` since the marcher is based on path and not a error matcher

Reply:
My thinking was that it's an error waiter acceptor because it leads the request to failure. I can see how it's confusing. NVM then!